frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -83,6 +135,10 @@ export const parseEncodedSnapshots = async (
             const snapshotLine = typeof l === 'string' ? (JSON.parse(l) as EncodedRecordingSnapshot) : l
             const snapshotData = isRecordingSnapshot(snapshotLine) ? [snapshotLine] : snapshotLine['data']
 
+            if (!isMobileSnapshots) {
+                isMobileSnapshots = hasAnyWireframes(snapshotData)
```

Comment:
Should we account for screenshot mode too?

Reply:
screenshot mode sends the screenshots as wireframes... 
wireframe mode doesn't send screenshots

🤯 🤣